README.md

```diff
@@ -27,62 +20,123 @@ Next, build from sources using:
 
 ## Running
 
-#### Server 0:
+### The config file
+The client and servers use a common configuration file, which contains the
+parameters for the system. The config file is also used to choose between the
+different modes of operation. Here, we show the basic structure of the config
+file. Each mode (Weighted Heavy Hitters, Attribute-Based Metrics, and Plain
+Metrics with Prio) uses a different config. The contents that are shared between
+all the config files are shown below:
+
+```bash
+{
```

Comment:
nit: This isn't copy-pastable because JSON doesn't like inline comments :(

I'd suggest leaving a copy-pastable version here, then clarifying the parameters below.

Note:  These days, TOML is the preferred format for configuration files. There is a serde implementation, so it would be relatively easy for us to replace JSON with TOML.

Reply:
OH nice, yeah let's merge this and I'll do a separate PR to replace it with TOML after we merge into main.